internal/util/projutil/exec.go

```diff
@@ -48,11 +48,11 @@ type GoCmdOptions struct {
 	Env []string
 	// Dir is the dir to run "go {cmd}" in; exec.Command.Dir is set to this value.
 	Dir string
-	// GoMod determines whether to set the "-mod=vendor" flag.
-	// If true, "go {cmd}" will use modules.
-	// If false, "go {cmd}" will not use go modules. This is the default.
+	// NoGoMod determines whether to set the "-mod=vendor" flag.
```

Comment:
What was the choice behind choosing `NoGoMod`? If the default is `GoMod` it would make more sense for me to keep it simple and go with `GoMod`, as we have to now below check if it is `not not go mod`. Think `GoMod` reads simpler, IMHO.

Reply:
I suppose using `GoMod` is fine if we're checking `projutil.IsDepManagerGoMod()`. If we ever stop supporting `dep` we can switch this but for now I'll revert to `GoMod`.

Reply:
@LiliC @estroz I think `NoGoMod` is better, but I think it's fine to leave it `GoMod` in this PR since we're explicitly setting it everywhere.

The idea is to structure your options struct fields such that the default zero-value of the field results in the desired default behavior. In our case we want go modules to be enabled by default, so if we were using `NoGoMod`, we would only have to explicitly set it to `true` when we wanted to disable go modules. Otherwise, it could be left unset (i.e. `false`) and we'd get go modules enabled implicitly.

Reply:
@joelanford while I agree with your logic, we have to check if modules are enabled/used for a project regardless since we still support `dep`. Therefore, either way makes sense to me.